lib/child_process.js

```diff
@@ -416,6 +416,11 @@ function normalizeSpawnArguments(file, args, options) {
   else if (options === null || typeof options !== 'object')
     throw new ERR_INVALID_ARG_TYPE('options', 'object', options);
 
+  // Make a shallow copy so we don't clobber the user's options object
+  // and we strictly make use of Object.create(null) so that only self
+  // properties are checked
```

Comment:
Tiny nit, but can you add a period at the end of the sentence. You can also update the comment on line 138 to say the same thing.

Reply:
Updated.